server/openapi_server/dbmodels/challenge.py

```diff
@@ -7,8 +7,8 @@
 
 class Challenge(BaseDocument):
     name = StringField(required=True, unique=True)
-    startDate = StringField(required=True, regex=r"^\d{4}(-\d{2})?(-\d{2})?$")
-    endDate = StringField(required=True, regex=r"^\d{4}(-\d{2})?(-\d{2})?$")
+    startDate = DateTimeField(required=True)
```

Comment:
Refer to PR comment.

Reply:
>Would having just a year be an issue if we move back to datetime?

Reply:
How many past challenges do not have day-month-year information?

Reply:
Most of the content in ROCC will come from future challenges, so they have more weights than past challenges. I would prefer to try getting more accurate information from past challenges than using this lack of information to make a design decision. If we can't find more information about a challenge, we could do 01-01-YYYY (I tried saving 00-00-YYYY in MongoDB for DateTime objects and it doesn't work).

Reply:
Luckily, there are only 3 dates that don't have a month and day. I'll see if I can revisit those challenges to get a more descriptive date.